components/feature/readerview/src/main/java/mozilla/components/feature/readerview/ReaderViewFeature.kt

```diff
@@ -35,10 +38,13 @@ class ReaderViewFeature(
     private val context: Context,
     private val engine: Engine,
     private val sessionManager: SessionManager,
+    controlsView: ReaderViewControlsView,
     private val onReaderViewAvailableChange: OnReaderViewAvailableChange = { }
 ) : SelectionAwareSessionObserver(sessionManager), LifecycleAwareFeature, BackHandler {
 
     private val config = Config(context.getSharedPreferences("mozac_feature_reader_view", Context.MODE_PRIVATE))
+    internal val presenter = ReaderViewControlsPresenter(controlsView, config)
+    internal val interactor = ReaderViewControlsInteractor(controlsView, config)
```

Comment:
nit: maybe we should rename these to `controlsPresenter` and `controlsInteractor` as well, since they are only for the controls UI?